src/VisualStudio/NuGet.Packaging.VisualStudio.15/Templates/Projects/Nuget/Package/Package.nuproj

```diff
@@ -14,6 +14,7 @@
 	</ItemGroup>
 	<PropertyGroup Label="Configuration">
 		<PackageId>$safeprojectname$</PackageId>
+		<PackageVersion>1.0.0</PackageVersion>
```

Comment:
I don't tend to see conditions like that in the project file itself. The user can freely change the value here, and this is during MSBuild's evaluation phase. So any Target, no matter where, can override it. And a .targets import could also override it. The only place that someone can't override it without this Condition you propose is a .props file. And for a .props file to set a value that overrides something in the project file would be quite odd.